packages/ag-charts-community/src/chart/agChartV2.ts

```diff
@@ -346,8 +346,6 @@ abstract class AgChartInternal {
             };
         }
 
-        await chart.awaitUpdateCompletion();
```

Comment:
NOTE: No longer needed as the only call to `updateDelta()` is to a code-block which has already acquired the `Chart.updateMutex()` lock so there are no other pending updated.